packages/same-structure/src/sameStructure.js

```diff
@@ -161,12 +165,13 @@ function pathStr(path) {
 // mustBeSameStructureInternal
 function mustBeSameStructureInternal(left, right, message, path) {
   function complain(problem) {
-    const template = harden([
-      `${message}: ${problem} at ${pathStr(path)}: (`,
-      ') vs (',
-      ')',
-    ]);
-    insist(false)(template, left, right);
+    assert.fail(
+      details(
+        [`${message}: ${problem} at ${pathStr(path)}: (`, ') vs (', ')'],
```

Comment:
This is the other place we're dynamically computing a normally static template, and then manually calling the `details` template tag as a function.